code/game/objects/structures/window.dm

```diff
@@ -57,13 +57,16 @@
 		shatter()
 	else
 		if(sound_effect)
-			playsound(loc, 'sound/effects/Glasshit.ogg', 100, 1)
+			playsound(loc, 'sound/effects/glass_hit.ogg', 100, 1)
 		if(health < maxhealth / 4 && initialhealth >= maxhealth / 4)
-			visible_message("[src] looks like it's about to shatter!" )
+			visible_message(span("danger", "[src] looks like it's about to shatter!"))
+			playsound(loc, "glasscrack", 100, 1)
```

Comment:
All these playsound here could be a bit excessive ingame (especially if combined with glass_hit)

Reply:
It is actually a quiet *crack* sound
and gives good audio feedback on how you're doing damage on the glass.
Anyway, it doesn't last more than ~1 second, so it won't do much overlap anyway.